conf/test.config

```diff
@@ -20,13 +20,13 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input  = "${projectDir}/assets/samplesheet.csv"
+    input               = "https://github.com/nf-core/test-datasets/raw/detaxizer/samplesheets/samplesheet.csv"
 
     // Genome references
-    fasta               = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta"
+    fasta               = "https://github.com/nf-core/test-datasets/raw/detaxizer/host_reference/genome.hg38.chr21_10000bp_region.fa"
```

Comment:
I dont know right now why, but I use `https://raw.githubusercontent.com/nf-core/test-datasets`

Reply:
Both formats seem to work, but `https://raw.githubusercontent.com/` looks nicer, changing that in the next commit